sources/packages/backend/libs/sims-db/src/entities/student-assessment.model.ts

```diff
@@ -286,6 +286,10 @@ interface BaseAssessment {
  * Interface for FullTime assessment payload.
  */
 export interface FullTimeAssessment extends BaseAssessment {
+  parentalVoluntaryContribution: number;
+  exceptionalCosts: number;
+  transportationCost: number;
```

Comment:
What was the need to update this object.? We update the interface only if it is required in the code to parse or validate the values.